projects/1.16.1/assets/astral-sorcery/astralsorcery/lang/zh_cn.json

```
@@ -162,161 +162,161 @@
   "astralsorcery.journal.research.brilliance": "辉煌",
 
   "astralsorcery.journal.node.WELCOME.name": "欢迎",
-  "astralsorcery.journal.WELCOME.1.text": "我们很高兴您能选择星空学院进行本科魔法学习。您手上的宝典是由学院的讲师为您在学院学习期间而特别准备的学习指南。  所有宝典都链接到我们的星能档案中，会随着您的学习进度自动扩展可用的资料。<NL>因此，您可以靠自学而不需要学院的教授来言传身教的指导。",
-  "astralsorcery.journal.WELCOME.2.text": "以最适合的进度来适配您的学习需求。<NL>但不要误以为您的学习会很轻松。与其他的学习机构相比，本学院以毕业生的优异成绩而自豪，您需要用宝典来证明对于星能操纵基本原理的了解，才能获准从星能档案中获得更多的资源倾斜。<NL>我们都祝您学习顺利！<NL>——星空学院全体教职员工。",
+  "astralsorcery.journal.WELCOME.1.text": "我们很高兴你能选择星辉学院进行本科魔法学习。你手上的宝典是由学院的讲师为你在学习期间而特别准备的学习指南。所有宝典都链接到了我们的星光档案中，会随着你的学习进度自动扩充可用的资料。<NL>如此的教学方法不再需要教授出面进行教导，",
+  "astralsorcery.journal.WELCOME.2.text": "并且你还能自行掌握学习节奏，更好地适应自身的学习需要。<NL>但不要误以为你的学习会很轻松。在一众学习机构中，本学院极为注重毕业生的素质，而且你还需要向宝典证明你对于星光操纵基本原理的了解，才能获准从星光档案中获得更多的资源倾斜。<NL>愿你学习之路一帆风顺！<NL>——星辉学院全体教职员工。",
 
-  "astralsorcery.journal.node.SHRINES.name": "圣陵学院",
-  "astralsorcery.journal.SHRINES.1.text": "该学院认为在建有神殿的地区附近有利于进行唯心研究。它们一般有两种变体：内藏有能进行星能操纵的浮动水晶的大型精致神殿和较小型的初学者遗迹。这两种类型的建筑内都存储有一部分的物资，当然其内的物品也会受上个使用该建筑的使用者而有所变化。大型的神殿通常位于没有树木遮挡的空旷区域，比如沙漠或者高山顶部。",
-  "astralsorcery.journal.SHRINES.2.text": "大型的神殿内含有可以用于聚焦和操纵星能的标准聚能水晶。所有人都能借此对五大明亮级别的星座之一进行调和共鸣，对于初学者来说在研究方面有极大的助益。<NL>虽然这些水晶本身是完美无瑕的，但它们是重新精制而成的，本质不够纯净，因此不要期望它与那些熟练掌握水晶调制技术者制作的聚能水晶有相同的星能量。这些水晶也因此无法进行性能强化。",
-  "astralsorcery.journal.SHRINES.3.text": "作为公共学习工具的浮动水晶周围设置了特殊的力场控制结界以防止提升者取走或修改它们，这保障了该区域所有提升者的重复利用性。<NL>该学院提醒所有提升者，星能是无法穿透所有不透明的固体物质。当学院需要关闭时，会进行顶部覆盖以防止学区受到恶劣天气和敌对生物的影响，如果要使用神殿内部的浮动水晶，记得移除顶部覆盖。",
+  "astralsorcery.journal.node.SHRINES.name": "学院祭坛",
+  "astralsorcery.journal.SHRINES.1.text": "某些地域矗立着一些祭坛，学院认为这些地方对于精神方面的研究颇有益处。祭坛一般有两种变体：内藏有能进行星光操纵的浮动水晶的大型精致祭坛，和较小型的学员祭坛。这两种类型的建筑内都存储有一部分的学习所必须的物资，当然其内的物品也会受上个使用该建筑的使用者而有所变化。大型的神殿通常位于没有树木遮挡的空旷区域，比如沙漠或者高山顶部。",
+  "astralsorcery.journal.SHRINES.2.text": "大型的神殿内含有可以用于聚焦和操纵星光的标准聚集水晶。所有人都能借此对五大“明亮级”的星座之一进行调和共鸣，这对于学员的研究有极大的助益。<NL>这些水晶虽经精雕细琢，但纯度却仍差一些火候，因此不要期望它能够提供与那些技术炉火纯青的水晶匠者制作的聚集水晶相同等级的星光量。也因如此，这些水晶无法在性能上更进一步了。",
+  "astralsorcery.journal.SHRINES.3.text": "作为公共学习工具，浮动水晶的周围已被设置了特殊的结界，因而学员无法取走或修改它们。这保障了该区域所有学员的能够重复使用这一工具。<NL>本学院提醒所有学员，星光是无法穿透不透明的固体物料。当学院需要关闭时会加装屋顶遮蔽，以防止学区受到恶劣天气和敌对生物的影响，如果要使用神殿内部的浮动水晶，记得移除屋顶遮蔽。",

```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "astralsorcery.journal.SHRINES.1.text": "某些地域矗立着一些祭坛，学院认为这些祭坛对于精神方面的研究颇有益处。祭坛一般有两种变体：大型而精致的祭坛，其中内藏有能进行星光操纵的浮动水晶；和较小型的学员祭坛。这两种建筑内都存储有部分学习所必需的物资，当然这些物品也会因这一建筑的上次使用而有所变化。大型的神殿通常位于没有树木遮挡的空旷区域，比如沙漠或者高山顶部。",
  "astralsorcery.journal.SHRINES.2.text": "大型的神殿内含有可以用于聚焦和操纵星光的标准聚集水晶。所有人都能借此对五大“明亮级”的星座之一进行调和共鸣，这对学员的研究有极大的助益。<NL>这些水晶虽经精雕细琢，纯度却仍差一些火候，因此不要期望它所提供的星光量能与那些技术炉火纯青的水晶匠者制作的聚集水晶位于相同等级。也因如此，这些水晶无法在性能上更进一步了。",
  "astralsorcery.journal.SHRINES.3.text": "作为公共学习工具，浮动水晶的周围已被设置了特殊的结界，因而学员无法取走或修改它们。这保障了该区域所有学员的能够重复使用这一工具。<NL>本学院提醒所有学员，星光无法穿透不透明的固体物料。学院在需要时会加装屋顶遮蔽，以防止学区受到恶劣天气和敌对生物的影响，如果要使用神殿内部的浮动水晶，记得移除屋顶遮蔽。",
```